core/services/ocr2/validate/validate.go

```diff
@@ -114,6 +114,8 @@ func validateSpec(tree *toml.Tree, spec job.Job) error {
 		return nil
 	case types.Mercury:
 		return validateOCR2MercurySpec(spec.OCR2OracleSpec.PluginConfig, *spec.OCR2OracleSpec.FeedID)
+	case types.GenericPlugin:
```

Comment:
Maybe we can change this from `plugin` to `generic`?

Reply:
(I'll follow up with this since it's part of chainlink-relay.